pages/blog/20190729-rpiandtreehousesremote.md

```diff
@@ -0,0 +1,84 @@
+# Initial Steps to Get Started with Treehouses
+Welcome to treehouses. This is our system allowing us to share our different learning platforms via the Rasperry Pi series of single-board computer devices. Here is a quick start for interacting your Raspberry Pi with Treehouses Remote.
+## Step 1
+Assuming you have an Android device and/or using an Android simulator; here is the way to go about it:
+Download the latest version of treehouses image [here](http://download.treehouses.io/) and burn it with [etcher](https://www.balena.io/etcher/). Once burned put your microsd card into your raspberry pi 
+## Step 2
+With your android device download [treehouses remote](https://play.google.com/store/apps/details?id=io.treehouses.remote). 
+## Step 3
+After installing it, get into your bluetooth settings and pair your system with your Raspberry Pi (optional).
+
+![bluetooth pairing](images/20190906-bluetooth-pairing.png) ![bluetooth paired](images/20190906-bluetooth-paired.png)
+
+## Step 4
+Once you're paired with your Raspberry Pi, open treehouses remote and connect to your Raspberry Ri.
+
+![treehouses remote start page](images/20190906-treehouses-remote-start-page.png) ![treehouses remote wm pg select device](images/20190906-treehouses-remote-wm-pg-select-device.png)
+
+![treehouses remote wm pg connecting device](images/20190906-treehouses-remote-wm-pg-connecting-device.png) ![treehouses remote wm pg connected device](images/20190906-treehouses-remote-wm-pg-connected-device.png)
+
+## Step 5
+At the terminal we'll get into setting up a bridge and setting up a WiFi
+
+![treehouses remote getting to terminal](images/20190906-treehouses-remote-getting-to-terminal.png) ![treehouses remote terminal](images/20190906-treehouses-remote-terminal.png)
+
+### Setting Up WiFi
+In your terminal enter `treehouses wifi <your local WiFi name> [your password]`
+
+![treehouses terminal keyboard](images/20190906-treehouses-terminal-keyboard.png)
+
+![treehouses wifi red](images/20190906-treehouses-wifi-red.png) ![treehouses wifi blue](images/20190906-treehouses-wifi-blue.png)
+
+treehousesWifi would be your wifi name 0123456789 woud be your password
```

Comment:
I feel like something should be done to have `treehousesWifi` and `0123456789` render differently (because they're from lines of code).